src/traces/choropleth/attributes.js

```diff
@@ -15,7 +15,7 @@ var extendFlat = require('../../lib/extend').extendFlat;
 
 var ScatterGeoMarkerLineAttrs = ScatterGeoAttrs.marker.line;
 
-module.exports = {
+module.exports = extendFlat({}, colorscaleAttrs, {
```

Comment:
Hmm. This will change the attribute ordering. 

Would something like:

```js
module.exports = extendFlat({}, {
  locations: {/* */},
  locationmode: {/* */},
  // ...
}, 
colorAttributes('z'), {
  hoverinfo: {/* */},
  _nestedModules: { /* */}
});
```

be too esoteric.